### PR TITLE
Adds jobs to seeds file

### PR DESCRIPTION
**Pull request by bplantico**
## Changes proposed in this pull request:
*Adds jobs seeds to seed file.

### Model Test Coverage: 100%
- [ ] Tested my new feature(s) as well as any feasible edge cases (if possible)
- [x] Checked coverage/index.html - did not add any new code that's not covered by testing (if possible)
- [x] Ran the test suite - all tests are passing
- [x] Merged in the latest master to my branch with git pull origin master & resolved merge conflicts
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -17,3 +17,14 @@
 garden.env_measurements.create!(soil_temperature: 63.88, soil_moisture: 73.90, created_at: 30.minutes.ago)
 garden.env_measurements.create!(soil_temperature: 65.55, soil_moisture: 71.02, created_at: 15.minutes.ago)
 garden.env_measurements.create!(soil_temperature: 69.33, soil_moisture: 68.86)
+
```
Comment (Mycobee) [inline db/seeds.rb]: Thanks for doing this.  I am working on job functionality with the pi currently.  

I think we can get around all this time param filtering with web sockets.
